vit-servicing-station-tests/src/tests/graphql/proposals.rs

```diff
@@ -13,8 +119,26 @@ pub fn get_proposals_by_id() {
         .proposal_id
         .parse()
         .unwrap();
-    assert!(server
-        .graphql_client_with_token(&snapshot.token_hash())
-        .proposal_by_id(proposal_id)
-        .is_ok());
+
+    let graphql_client = server.graphql_client_with_token(&snapshot.token_hash());
+
+    get_proposal_by_id_test(&graphql_client, proposal_id, &snapshot);
+
+    run_query_with_comment_test(&graphql_client, proposal_id);
+
+    additional_field_test(&graphql_client, proposal_id);
+
+    aliases_test(&graphql_client, proposal_id);
+
+    alias_test(&graphql_client, proposal_id);
+
+    cycle_test(&graphql_client, proposal_id);
+
+    proposal_without_arguments_test(&graphql_client);
+
+    proposals_test(&graphql_client, &snapshot);
+
+    proposal_with_fragments_test(&graphql_client, proposal_id);
```

Comment:
I think it would be better to have this tests completely separated, one in each `#[test]` case.